### ISSUE TITLE
Disable the owner API secret doesn't work

### ISSUE THREAD

**Issue by garyyu** [mimblewimble/grin-wallet#93]
To disable the wallet API secret, we need comment `api_secret_path`, but it doesn't work, we have to disable both `api_secret_path` and `node_api_secret_path`:
```
#path of the secret token used by the API to authenticate the calls
#comment it to disable basic auth
#api_secret_path = "/Users/garyyu/.grin/floo/.api_secret"

#location of the node api secret for basic auth on the Grin API
#node_api_secret_path = "/Users/garyyu/.grin/floo/.api_secret"
```

**Comment by yeastplume** [comment]
Is this still a problem?

**Comment by yeastplume** [comment]
Fixed in #218
